templates/kafka/docker-compose.yml

```diff
@@ -0,0 +1,27 @@
+version: "2"
+services:
+    zookeeper:
+      image: docker.io/bitnami/zookeeper:3.8
+    ports:
+      - "2181:2181"
+    volumes:
+      - "zookeeper_data:/bitnami"
+    environment:
+      - ALLOW_ANONYMOUS_LOGIN=yes
```

Comment:
syntax errors here, cant bring up the containers

![image](https://user-images.githubusercontent.com/13061926/222965497-6aecb1ae-9e12-40cd-9be0-086ba2e3f362.png)

there's a missing indentation from line 5 to 10 and `environment` should be a mapping (`- env_key: value`) instead of `- env_key=value`

Reply:
just fixed :)